### DISCUSSION TITLE
Finding the Angle of the Lowest Point in the Sky

### DISCUSSION THREAD

**Discussion by infinite-happiness** [cosinekitty/astronomy#354] [Q&A]
Hello!
I was looking for the astronomical dawn in Karlstad, Sweden a certain date this summer, using this code:
`Astronomy.SearchAltitude('Sun', observer, +1, date, 1, -18);`

If I understand correctly, astronomical dawn does not occur for the given date. I suppose the sun never gets to that angle.

Now I was able to find the time when the lowest point occured:
`Astronomy.SearchHourAngle('Sun', observer, 12, date, +1);`

However, since this is looking for the lowest point, the angle is not known. For curiosity, I would like to know the angle in terms of understanding how close it is to astronomical/nautical/civicl dawn, and from reading other posts the hour angle is a different type of angle.

So perhaps the function SearchHourAngle should also return hour angle and altitude angle?

**Comment by lulunac27a** [comment]
To get the altitude of the object from `SearchHourAngle`, use `Astronomy.SunAltitude(lowestAltitudeTime, observer)` for the sun or `Astronomy.Horizon(lowestAltitudeTime, observer, object.ra, object.dec).altitude` where `lowestAltitudeTime = Astronomy.SearchHourAngle('Sun', observer, 12, date, +1);`.
